### PR TITLE
perf: speedup drive tests

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

Drive tests were running sequentially, making it extremely slow on large networks.

## What was done?

Parallelize tests

## How Has This Been Tested?

On testnet (and devnet-jack-daniels with alternate echo string `0a0a080a0668656c6c6f21`

## Breaking Changes

Won't work with Drive 0.23, separate PR necessary

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: test/smoke/drive.js [hunk 1]
```diff
@@ -28,32 +28,37 @@ async function sendEcho(ip) {
...
-        }
+      return Promise.all(promises).catch(() => Promise.resolve());
+    });
 
-        // eslint-disable-next-line no-underscore-dangle
-        await sendEcho(inventory._meta.hostvars[hostName].public_ip);
+    for (const hostName of allHosts) {
+      describe(hostName, () => {
+        it('should listen for ABCI connection', () => {
+          expect(echoInfo[hostName]).to.be.true();
```
Comment (shumkov) [inline test/smoke/drive.js:59]: Will we see the error in this case?

Comment (strophy) [reply test/smoke/drive.js:59]: Yes, when drive fails to respond to the ping we get:
```
  1) Smoke
       Drive
         All nodes
           masternode-2
             should listen for ABCI connection:
     AssertionError: expected [Error: Can't connect to ABCI port: timeout.] to be true
      at Context.<anonymous> (test/smoke/drive.js:59:48)
      at processImmediate (node:internal/timers:466:21)
```

Is that good enough? Or should I collect errors in a different object?

**Review by shumkov — APPROVED** [review]
👍

File: test/smoke/drive.js (hunk 1 above)
Comment (shumkov) [reply test/smoke/drive.js]: I would recommend using `expect.fail` as a more idiomatic way.

**Pull request merged**